packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -0,0 +1,179 @@
+import {Component, h, Prop, Element, Listen} from '@stencil/core';
+import {applyFocusVisiblePolyfill} from '../../../utils/initialization-utils';
+import {
+  DisplayConfig,
+  ResultContextEvent,
+} from '../../search/result-template-components/result-template-decorators';
+import {
+  ResultLayout,
+  ResultDisplayDensity,
+  ResultDisplayImageSize,
+  ResultDisplayLayout,
+} from '../../common/layout/display-options';
+import {RecsResult} from '..';
+import {AtomicRecsStore} from '../atomic-recs-interface/store';
+import {ResultRenderingFunction} from '../../common/result-list/result-list-common-interface';
+
+/**
+ * The `atomic-recs-result` component is used internally by the `atomic-recs-list` component.
+ */
+@Component({
+  tag: 'atomic-recs-result',
+  styleUrl: '../../common/result/result.pcss',
+  shadow: true,
+})
+export class AtomicRecsResult {
+  private layout!: ResultLayout;
+  private resultRootRef?: HTMLElement;
+  private executedRenderingFunctionOnce = false;
+  @Element() host!: HTMLElement;
+
+  /**
+   * Whether an atomic-result-link inside atomic-recs-result should stop propagation.
+   */
+  @Prop() stopPropagation?: boolean;
+
+  /**
+   * The result item.
+   */
+  @Prop() result!: RecsResult;
+
+  /**
+   * Global state for Atomic.
+   * @internal
+   */
+  @Prop() store?: AtomicRecsStore;
+
+  /**
+   * The result content to display.
+   */
+  @Prop() content?: ParentNode;
+
+  /**
+   * How results should be displayed.
+   */
+  @Prop() display: ResultDisplayLayout = 'list';
+
+  /**
+   * How large or small results should be.
+   */
+  @Prop() density: ResultDisplayDensity = 'normal';
+
+  /**
+   * How large or small the visual section of results should be.
+   *
+   * This may be overwritten if an image size is defined in the result content.
+   */
+  @Prop() imageSize: ResultDisplayImageSize = 'icon';
+
+  /**
+   * Classes that will be added to the result element.
```

Comment:
```suggestion
   * The classes to add to the result element.
```
Also, I'm guessing it's a comma separated string? Like `"class1,class2,class3"`? Maybe it'd be worth giving an example? 🙂

Reply:
No it's simply a string that's added to the class html property. Similar to the atomic-result, I'll modify the docs in these as well!